database/factories/Forum/PollOptionFactory.php

```diff
@@ -16,6 +19,7 @@ public function definition(): array
     {
         return [
             'poll_option_text' => fn () => $this->faker->sentence(),
+            'topic_id' => Topic::factory()->poll(0),
```

Comment:
set to `null` so `PollOption` can't be created without `Topic`, it doesn't make sense to factory `PollOption` by itself.